client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-processing-loading-messages.tsx

```diff
@@ -18,6 +22,21 @@ export function useProcessingLoadingMessages( flow?: string | null ): LoadingMes
 		[]
 	);
 
+	if ( flow && isNewHostedSiteCreationFlow( flow ) ) {
+		return [ { title: __( 'Creating your site' ), duration: Infinity } ];
+	}
+
+	if ( flow && isTransferringHostedSiteCreationFlow( flow ) ) {
+		return [
+			{ title: __( 'Laying the foundations' ), duration: 3500 },
+			{ title: __( 'Warming up CPUs' ), duration: 2000 },
+			{ title: __( 'Installing WordPress' ), duration: 2000 },
+			{ title: __( 'Securing your data' ), duration: 3500 },
+			{ title: __( 'Distributing your site worldwide' ), duration: 3500 },
+			{ title: __( 'Closing the loop' ), duration: 5000 },
```

Comment:
As the Atomic transfer process takes a fair bit of time, I think we can extend these durations by a factor of 1.5. Let me know what you think.

Reply:
I think so also, closing the loop happens too fast. I would rather transfer completes and that statement is shown rather than it shown multiple times.